testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -625,3 +625,44 @@ def test_auto_is_unscaled_match(self, universes):
         for ts_a, ts_s in zip(universes["auto"].trajectory,
                               universes["unscaled"].trajectory):
             assert_almost_equal(ts_a.positions, ts_s.positions, decimal=5)
+
+
+class TestLammpsTriclinic(object):
+    @pytest.fixture()
+    def u_dump(self):
+        return mda.Universe(LAMMPSDUMP_triclinic, format='LAMMPSDUMP',
+                            lammps_coordinate_convention="auto")
+
+    @pytest.fixture()
+    def u_data(self):
+        return mda.Universe(LAMMPSdata_triclinic, format='data',
+                            atom_style='id type x y z')
+
+    @pytest.fixture()
+    def reference_box(self):
+        # manually copied from triclinic data file
+        xlo = -0.32115478301032807
+        xhi = 16.831069399898624
+        ylo = -0.12372358703610897
+        yhi = 25.95896427399614
+        zlo = -0.045447071698045266
+        zhi = 12.993982724334792
+        xy = 1.506743915478767
+        xz = -6.266414551929444
+        yz = -0.42179319547892025
+
+        box = np.zeros((3, 3), dtype=np.float64)
+        box[0] = xhi - xlo, 0.0, 0.0
+        box[1] = xy, yhi - ylo, 0.0
+        box[2] = xz, yz, zhi - zlo
+
+        return mda.lib.mdamath.triclinic_box(*box)
+
+    def test_box(self, u_dump, u_data, reference_box):
```

Comment:
I like the threefold validation here with a regression test, really good idea. Perhaps just leave a note that this tests both the `data` and `dump` reader.